Simperium/build.gradle

```diff
@@ -68,54 +53,25 @@ android {
     useLibrary 'org.apache.http.legacy'
 }
 
-task versionConfig(group: "build", description: "Generate version class") {
-    def dir = "${buildDir}/generated/source/version"
-    def file = new File(dir, "com/simperium/Version.java")
-
-    android.sourceSets.main.java.srcDir dir
-
-    doLast {
-        def describe = gitDescribe()
-
-        logger.info "Version: ${describe}"
-
-        file.getParentFile().mkdirs()
-        def writer = new FileWriter(file)
-        writer.write("""/* auto-generated file do not modify */
-package com.simperium;
-
-public final class Version {
-
-    public static final String LIBRARY_NAME = "android";
-
-    public static final String NAME = "android-${project.version}";
-
-    /* project.version */
-    public static final String NUMBER = "${project.version}";
-
-    /* git rev-parse --short --verify HEAD */
-    public static final String BUILD = "${gitHash()}";
-
-    /* git describe --always --dirty=-dirty */
-    public static final String DESCRIBE = "${gitDescribe()}";
+tasks.withType(Javadoc) {
+    options.addStringOption('Xdoclint:none', '-quiet')
+    options.addStringOption('encoding', 'UTF-8')
 }
-""")
-        writer.close()
+
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+                from components.clientRelease
+                groupId "com.automattic"
```

Comment:
I picked `com.automattic` as the domain. We can change it to `simperium` and the `artifactId` to `simperium-android` if it's preferred. This decision changes how the dependency will be added to the projects, [like so](https://github.com/Automattic/simplenote-android/pull/1446/files#diff-43f4c0dc2dc319470a77750c75fc377f96ac31e7183e84fac0ee5ada14aa1042R84).